tensorflow/python/data/util/random_seed_test.py

```diff
@@ -18,18 +18,21 @@
 from __future__ import division
 from __future__ import print_function
 
+from absl.testing import parameterized
+
 from tensorflow.python.data.util import random_seed as data_random_seed
 from tensorflow.python.eager import context
+from tensorflow.python.data.kernel_tests import test_base
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import random_seed
-from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 
 
-class RandomSeedTest(test.TestCase):
+class RandomSeedTest(test_base.DatasetTestBase, parameterized.TestCase):
 
-  @test_util.run_in_graph_and_eager_modes
+  @combinations.generate(test_base.default_test_combinations())
```

Comment:
The `test_cases` variable below should be moved to the combinations definition instead. In other words, instead of having a single test that runs a for all for different parameters combinations, we should be running one test for each combination (and that's what the `combinations.generator` decorator is used for).

Reply:
This comment applies to all other tests that use the `test_cases` pattern.

Reply:
addressed in the commit.